### PR TITLE
Housekeeping dmn-moddle

### PR DESCRIPTION
**Pull request by MaxTru**
* Migrate to GHA
* Bump dependencies and resolve security audit warnings
* Fix outdated DMN references (1.1 -> 1.3)

### CODE REVIEW

**Review by barmac — APPROVED** [review]
LGTM

File: .npmignore [hunk 1]
```diff
@@ -5,5 +5,4 @@ lib/
 !dist/
```
Comment (nikku) [inline .npmignore:5]: Should we add the `.github` folder here? Otherwise we'll leak our CI configuration into published distributions of this library.

Comment (MaxTru) [reply .npmignore:5]: absolutely! Will follow-up

Comment (MaxTru) [reply .npmignore:5]: adfa6ad8e69b60250b417473781f6b68744e87ba

**Pull request merged**